content/content-strategy/define-user-content-needs/create-actionable-artefacts/user-job-stories/index.yml

```diff
@@ -0,0 +1,27 @@
+pagetitle: User and job stories
+description: How to write user stories to create digital content.
+weight: 10
```

Comment:
Check this value against the others in the category. Should be unique. Increment in values of 10.

Reply:
Steve. Done.